test-integ/peering_commontopo/ac5_1_test.go

```diff
@@ -0,0 +1,245 @@
+		Service: NewFortioServiceWithDefaults(
+			clu.Datacenter,
+			clientSID,
+			func(s *topology.Service) {
+				s.EnvoyAdminPort = 0
+				s.DisableServiceMesh = true
+			},
+		),
+		Config: &api.ServiceConfigEntry{
+			Kind:      api.ServiceDefaults,
+			Name:      clientSID.Name,
+			Partition: ConfigEntryPartition(clientSID.Partition),
+			Protocol:  "http",
+		},
+	}
+	clientNode := ct.AddServiceNode(clu, client)
+
+	server := serviceExt{
+		Service: NewFortioServiceWithDefaults(
+			peerClu.Datacenter,
+			serverSID,
+			nil,
+		),
+		Config: &api.ServiceConfigEntry{
+			Kind:      api.ServiceDefaults,
+			Name:      serverSID.Name,
+			Partition: ConfigEntryPartition(serverSID.Partition),
+			Protocol:  "http",
+		},
+		Exports: []api.ServiceConsumer{{Peer: cluPeerName}},
+		Intentions: &api.ServiceIntentionsConfigEntry{
+			Kind:      api.ServiceIntentions,
+			Name:      serverSID.Name,
+			Partition: ConfigEntryPartition(serverSID.Partition),
+			Sources: []*api.SourceIntention{
+				{
+					Name:   client.ID.Name,
+					Peer:   cluPeerName,
+					Action: api.IntentionActionAllow,
+				},
+			},
+		},
+	}
+
+	serverNode := ct.AddServiceNode(peerClu, server)
+
+	s.clientSID = clientSID
+	s.serverSID = serverSID
+	s.nodeServer = serverNode.ID()
+	s.nodeClient = clientNode.ID()
+}
+
+func (s *serviceMeshDisabledSuite) test(t *testing.T, ct *commonTopo) {
+	dc := ct.Sprawl.Topology().Clusters[s.DC]
+	peer := ct.Sprawl.Topology().Clusters[s.Peer]
+	apiClient := ct.APIClientForCluster(t, dc)
+	peerName := LocalPeerName(peer, "default")
+
+	serverSVC := peer.ServiceByID(
+		s.nodeServer,
+		s.serverSID,
+	)
+
+	ct.Assert.HealthyWithPeer(t, dc.Name, serverSVC.ID, peerName)
+	s.testExportedServiceInCatalog(t, ct, apiClient, peerName)
+	s.testProxyDisabledInDC2(t, apiClient, peerName)
+	s.testSingleQueryFailover(t, apiClient, ct, peerName)
+}
+
+func (s *serviceMeshDisabledSuite) testExportedServiceInCatalog(t *testing.T, ct *commonTopo, apiClient *api.Client, peer string) {
+	t.Run("service exists and is healthy in catalog", func(t *testing.T) {
+		libassert.CatalogServiceExists(t, apiClient, s.clientSID.Name, nil)
+		libassert.CatalogServiceExists(t, apiClient, s.serverSID.Name, nil)
+		libassert.CatalogServiceExists(t, apiClient, s.serverSID.Name, &api.QueryOptions{
+			Peer: peer,
+		})
+		require.NotEqual(t, s.serverSID.Name, s.Peer)
+	})
+}
+
+func (s *serviceMeshDisabledSuite) testProxyDisabledInDC2(t *testing.T, cl *api.Client, peer string) {
+	t.Run("service mesh is disabled for HTTP Service in clients", func(t *testing.T) {
+		expected := fmt.Sprintf("%s-sidecar-proxy", s.clientSID.Name)
+		services := getServicesInCluster(t, cl, &api.QueryOptions{
+			Peer: peer,
+		})
+		require.NotContains(t, services, expected, fmt.Sprintf("error: should not create proxy for service: %s", services))
+	})
+}
+
+func (s *serviceMeshDisabledSuite) testSingleQueryFailover(t *testing.T, c *api.Client, ct *commonTopo, peer string) {
+	if s.DC == "dc2" {
```

Comment:
We should probably fix this before merge, but if not, I'd say it's better to just exclude that case from `serviceMeshDisabledSuites`

Reply:
Also, you can put a message in `t.Skip("TODO")` :)

Reply:
It's designed this way, test will not pass when running dc2->dc1 seeing as the  service in dc1 is down, therefore failover is not required and request will succeed.